tools/test/examples/update.py

```diff
@@ -212,6 +212,12 @@ def upgrade_example(github, example, tag, user, ref):
                 os.chdir(cwd)
                 return False
 
+        checkout_branch = ['git', 'checkout', branch]
```

Comment:
We should have the option to create a branch if it doesn't already exist. Also the option to force push directly to that branch on the upstream repo rather than creating a PR. No real need to raise a PR for OOB for example?